game-core/src/main/java/games/strategy/triplea/ui/UserActionText.java

```diff
@@ -83,6 +97,14 @@ public String getNotificationFailureOthers(final String actionKey) {
     return getMessage(actionKey, OTHER_NOTIFICATION_FAILURE);
   }
 
+  public String getNotificationFailureTarget(final String actionKey) {
+    if (hasMessage(actionKey, TARGET_NOTIFICATION_FAILURE)) {
+      return getMessage(actionKey, TARGET_NOTIFICATION_FAILURE);
+    } else {
+      return getNotificationFailureOthers(actionKey);
```

Comment:
Not following this logic. Shouldn't this return "NONE"?

Reply:
See above.